homeassistant/components/poolsense/sensor.py

```diff
@@ -0,0 +1,112 @@
+"""Sensor platform for the PoolSense sensor."""
+import logging
+
+from homeassistant.const import ATTR_ATTRIBUTION
+from homeassistant.helpers.entity import Entity
+
+from . import get_coordinator
+from .const import ATTRIBUTION
+
+_LOGGER = logging.getLogger(__name__)
+
+SENSORS = {
+    "Chlorine": {"unit": "mV", "icon": "mdi:pool", "name": "Chlorine"},
```

Comment:
not sure this correct unit?

Reply:
I agree that the unit is not the unit for chlorine but the device itself, to keep cost down uses some other method to measure chlorine relative to some other metric. As such the level is reported in millivolts. I considered changing it but if you use the supported app they use the unit mV.